staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -560,6 +571,10 @@ func AnonymousClientConfig(config *Config) *Config {
 		Burst:              config.Burst,
 		Timeout:            config.Timeout,
 		Dial:               config.Dial,
+		// ProxyURL can contain usernames and passwords to socks5 proxies, but we
+		// already preserve them when they are passed via the environment. This
+		// keeps the behavior of the explicit ProxyURL configuration consistent.
```

Comment:
add something related to this to the godoc for the method

Reply:
Obsolete.